packages/feedback-react/src/types.ts

```diff
@@ -0,0 +1,6 @@
+export type FeedbackOptions = 1 | 2 | 3 | 4 | 5;
```

Comment:
Litt usikker på navngivingen her.. `FeedbackOptions` høres ut som den skal inneholde innstillinger for komponenten. Hva med bare `FeedbackValue`? Da får du f.eks. `const score: FeedbackValue = 4`, og i komponent-interfacet `feedbackOptions: FeedbackValue[];` eller `feedbackOptions: Array<FeedbackValue>;`.

```suggestion
export type FeedbackValue = 1 | 2 | 3 | 4 | 5;
```

Reply:
Blir litt tydeliger ja